File: JS/canVote-2.js
function canVote(age) {
    if (age != null) {
        if (age < 18) {
            return "Cannot vote, Sorry!"
        } else {
            return "You can vote!"
        }
    }

    return "Oops!, age is not passed"
}

console.log(canVote(null));
console.log(canVote(17));
console.log(canVote(18));
console.log(canVote(28));
console.log(canVote(10));

File: Python/canVote-2.py
def can_vote(age):
    if age is not None:
        if age < 18:
            return "Cannot vote, sorry!"
        else:
            return "You can vote!"
    return "Oops!, age is not passed"

print(can_vote(None))
print(can_vote(17))
print(can_vote(18))
print(can_vote(28))
print(can_vote(10))
